backend/scripts/insert_photos.py

```
@@ -0,0 +1,107 @@
+            "created_at": now,
+            "updated_at": now,
+            "attachments": [],
+            "metadata": {"duration": 30} # Mock duration
+        })
+    
+    print(f"Inserting {len(entries)} items (10 photos, 2 videos, 2 audios) for user: {user_id}...")
+    
+    try:
+        response = supabase.table("entries").insert(entries).execute()
+        print(f"Successfully inserted {len(response.data)} media entries!")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)